src/index.html

```diff
@@ -1,14 +1,91 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <img
+        src="./images/logo.png"
+        alt="MOYO logo"
+        class="img">
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link is-active"
+                href="#">
+                APPLE
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#">
+                SAMSUNG
+              </a>
```

Comment:
коли атрибути переносяться то закриваюча дужка має бути також з нової лінії і без відступу
```suggestion
              <a
                class="nav__link"
                href="#"
              >
                SAMSUNG
              </a>
```

Reply:
@Denchepornyuk fixed